docs/content/config-schema.rst

```diff
@@ -1,10 +1,11 @@
-Config
-======
+FDB Config File
+---------------
 
 Config files define a number of parameters for the FDB. 
```

Comment:
Fundamentally, I think we need to have a section describing FDB concepts, semantic metadata, what the schema is and what it does, etc. before we get to this bit.

And generally this is simultaneously too chatty and too brief.